programs/survival/cam.sc

```diff
@@ -5,7 +5,11 @@
 // places spectating players without config in a safe places
 // and adds timeout for the player so they cannot switch instantly
 // change setting below to change that behaviour (0 to disable timeout)
-global_survival_timeout = 3;
+global_survival_timeout = 0;
```

Comment:
Y changed to 0 here? Is it cos old default is 0?

Reply:
Forgot to change back while testing. There was a discussion about changing it to zero tho in discord. MIght leave it like this if people agree.